src/ralph/data_center/forms/network.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from django import forms
+
+
+class NetworkInlineFormset(forms.models.BaseInlineFormSet):
+
+    def clean(self):
+        is_management = []
+        for form in self.forms:
+            cleaned_data = form.cleaned_data
+            if (
+                cleaned_data and
+                not cleaned_data.get('DELETE', False)
+            ):
+                is_management.append(cleaned_data.get('is_management'))
+
+        if is_management.count(True) > 1:
```

Comment:
`count` on empty list also works so:
`if  is_management.count(True) != 1`
is enough, what do you think?